### PR TITLE
Assignment 5

### PR DESCRIPTION
**Pull request by DominicAlvarez**
Backend is setup, need to work on front end components
website link:
https://victorious-bay-0650cb41e.3.azurestaticapps.net/
Api Link:
https://wordle-assignment5-app.azurewebsites.net/Word
https://wordle-assignment5-app.azurewebsites.net/Word/WordOfTheDay

### CODE REVIEW

**Comment by dpeterson8** [comment]
Create a page named "WordEditor" that allows words to be edited ❌✅
Support deleting a word ❌
Support adding a new word ❌
Editing words is not supported ✅
Duplicate words should not be allowed ❌✅
Support changing the common word boolean flag ❌
Words should be sorted alphabetically ✅
Provide a text box for searching words AS YOU TYPE which supports searching for words starting with letters specified ✅
Provide a menu item for navigating to the WordEditor page 
Anyone can look at the word list ✅
Word list is paginated (10-100 per page) ✅
Any logged in user can change the common word flag ❌
There is a login page/component/popup/etc for users to sign in ✅
Only users over 21 years of age (based on birthday) with a claim of MasterOfTheUniverse can add and remove words ❌
Users can be set up at start up in the API project as a seed (apriori), there does not need to be a sign up mechanism ✅

File: wordle-web/src/views/WordEditorView.vue
Comment (mmoran314) [inline wordle-web/src/views/WordEditorView.vue:1]: Would add the ability to navigate between pages rather than just having the 10-100 search results.

**Review by mmoran314 — COMMENTED** [review]
Create an editor for the available words ✅
Create a page named "WordEditor" that allows words to be edited ❌✅
Support deleting a word ❌
Support adding a new word ❌
Editing words is not supported ✅
Duplicate words should not be allowed ❌✅
Support changing the common word boolean flag ❌
Words should be sorted alphabetically ✅
Provide a text box for searching words AS YOU TYPE which supports searching for words starting with letters specified ✅
Provide a menu item for navigating to the WordEditor page ✅
Anyone can look at the word list ✅
Word list is paginated (10-100 per page) ❌✅
Seems as there is only one page but 10-100 are displayed as required
Any logged in user can change the common word flag ❌
There is a login page/component/popup/etc for users to sign in ✅
Only users over 21 years of age (based on birthday) with a claim of MasterOfTheUniverse can add and remove words ❌
Users can be set up at start up in the API project as a seed (apriori), there does not need to be a sign up mechanism ✅

Looks good, the word editor came out looking nice and just needs a few features added.
There is also the claims which seem pretty tricky from what we went over in class. 
Keep it up though!

File: Wordle.Api/Wordle.Api/Controllers/WordController.cs [hunk 1]
```diff
@@ -35,6 +36,12 @@ public async Task<Word> AddWord(string newWord, bool isCommon)
             return await _wordService.AddWordAsync(newWord, isCommon);
         }
 
+        [HttpDelete("Delete")]
```
Comment (meghanmae) [inline Wordle.Api/Wordle.Api/Controllers/WordController.cs:39]: This, as well as your other endpoints, need a policy

**Pull request closed without merging**